R/eesapi_url.R

```diff
@@ -0,0 +1,91 @@
+#' eesapi_url
+#'
+#' @param api_version EES API version
+#' @param endpoint Name of endpoint, can be get-summary, get-meta, get-data, query-data
+#' @param dataset_id ID of dataset to be connected to
+#' @param dataset_version Verson of dataset to be connected to
```

Comment:
data set is technically two words when in a sentence I think? (or even if not, it will be better to agree a convention now)